### PR TITLE
Added line clamp logic css to simple list

### PR DESCRIPTION
**Pull request by brentmiller1973**
## Description
Simple list cut off on Mobile views

## Jira Ticket
[- [TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TMEDIA-651)

## Acceptance Criteria
Acceptance Criteria:
- Restrict content to three lines
- Remove height restrictions 

## Test Steps

1. Checkout this branch `git checkout tmedia-651-list-cutoff`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/simple-list-block`
3. Open the published page of `Test - JC - All Blocks`
4. Inspect the text of the first simple list: 
image
5. Replace the text in `h3.simple-list-headline-text` With: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.`
6. Enable responsive mode and select `Pixel 5` as thee target device.
7. Ensure on all breakpoints (especially mobile) that text is not being cut off and ellipses are being applied 3 lines or more.

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x ] Confirmed all the test steps a reviewer will follow above are working.
- [x ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x ] Confirmed this PR has unit test files
  - [ x] Ran `npm run test`, made sure all tests are passing

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

**Review by matthewroach — DISMISSED** [review]
Looks like this is only working when the height of the headline is only able to fit 3 lines of text on, when the viewport is in between sizes and the space for the headline allows for more than three lines of text the ellipsis is shown along with the additional text

We will need to add another breakpoint for chromatic to catch this issue

Microsoft Edge-2022-02-08-10-42 30

File: blocks/simple-list-block/features/simple-list/simple-list.scss [hunk 1]
```diff
@@ -45,13 +45,19 @@
...
       text-decoration: none;
 
-      .simple-list-headline-text {
+      .simple-list-headline-text { /* stylelint-disable-line */
         font-size: calculateRem(16px);
         font-weight: normal;
         line-height: calculateRem(24px);
-        max-height: 72px;
         overflow: hidden;
+        /*
```
Comment (ecarlisle) [inline blocks/simple-list-block/features/simple-list/simple-list.scss]: Even though they are multiline comments, should be use multiple single-line comments w/ `//`?
Assuming those will be stripped out when compiled, making the CSS smaller while preserving the documentation internally?

Comment (brentmiller1973) [reply blocks/simple-list-block/features/simple-list/simple-list.scss]: Thanks @ecarlisle , assumed stylint had to have the /* style comments, but that is not the case.

**Comment by brentmiller1973** [comment]
> Looks like this is only working when the height of the headline is only able to fit 3 lines of text on, when the viewport is in between sizes and the space for the headline allows for more than three lines of text the ellipsis is shown along with the additional text
> 
> We will need to add another breakpoint for chromatic to catch this issue
> 
> Microsoft Edge-2022-02-08-10-42 30

@matthewroach Used fit-content on the height to resolve the issue.  Did check that fit-content is supported on our browser list and also manually verified on Chrome, Safari, FF, Edge.  Also updated the simple list story to include an extra long headline to proof out the solution.

**Review by nschubach — APPROVED** [review]
Looks like it's working in multiple browsers (Chrome, Firefox, Safari) and I don't see the issue Matt described anymore.

**Comment by JackHowa** [comment]
@visionbegin can you also update the baselines with storybook? if there was a deliberate change. this is setup to prevent style regressions https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2341536211/Storybook

Screen Shot 2022-02-15 at 15 34 34

**Pull request merged**